### DISCUSSION TITLE
CLI Debug Information

### DISCUSSION THREAD

**Discussion by wilson-webdev** [chakra-ui/chakra-ui#5008] [Q&A]
Does Chakra provide more information about errors using `chakra-cli tokens`? This is the output I get, but not sure where I should start to debug the issue.

```console
❯ chakra-cli tokens theme.ts

 Chakra UI CLI  v1.5.3 by chakra UI
Generate theme typings for autocomplete


✖ An error occurred
SyntaxError: Invalid or unexpected token
```

**Comment by amritk** [comment]
Did you ever figure this out?

**Reply by wilson-webdev** [reply]
I believe it was to do with the fact that we had colours nested more than one level e.g. `risk.high.50` instead of just `riskHigh.50`
